### ISSUE TITLE
something about dropout when testing...

### ISSUE THREAD

**Issue by Jackie-Chou** [kratzert/finetune_alexnet_with_tensorflow#19]
I'm studying deep learning and as I learn it, dropout rate when less than 1, say 0.5 for example, then during testing it's traditional to keep all the units and multiply the outputs by 0.5 in order to make a good inference. However, I'm not sure if it's my unawareness or just a bug, it seems that the mechanism has not been implemented, I hope you can check it.

**Comment by kratzert** [comment]
Fortunately all the scaling is done by TensorFlow behind the scene. All you have to do is to pass a keep probability of 1 to TensorFlow during inference and it will automatically scale all activations correctly for you.
